src/demo/models/pond.js

```diff
@@ -5,10 +5,9 @@ import constants, {ClassType, AppMode} from '../constants';
 
 export const init = async () => {
   const state = getState();
-  let fishWithConfidence = await predictAllFish(state);
-  fishWithConfidence = _.sortBy(fishWithConfidence, ['confidence']);
+  let fishWithPredictions = await predictAllFish(state);
   const fishByClassType = _.groupBy(
-    fishWithConfidence,
+    fishWithPredictions,
```

Comment:
should we still sort the items by confidence?

Reply:
My opinion is that there's a benefit in just showing a random sample of the fish, instead of the most confident, since it gives a more accurate representation of how well the model is doing, and more opportunities to see misclassified objects and debug them. I could see an argument that we want to show the top results to make the pond look better, though. Maybe it'd be useful to get Mike's input?

Reply:
that's a really good point. i think i agree with you. unless there's a good reason to sort them, it's probably better to leave as-is anyways